src/components/transport_manager/src/transport_adapter/threaded_socket_connection.cc

```diff
@@ -65,7 +65,7 @@ ThreadedSocketConnection::ThreadedSocketConnection(
 ThreadedSocketConnection::~ThreadedSocketConnection() {
   SDL_AUTO_TRACE();
   Disconnect();
-  thread_->join();
+  thread_->join(true);
```

Comment:
@orhan-mehmedov  is is possible to use enum value FORCE_STOPor JOIN against bool value?